riscv/execute.cc

```diff
@@ -165,26 +165,23 @@ inline void processor_t::update_histogram(reg_t UNUSED pc)
 // This is expected to be inlined by the compiler so each use of execute_insn
 // includes a duplicated body of the function to get separate fetch.func
 // function calls.
```

Comment:
Is this comment still accurate? At least, the function name needs updating

Reply:
Updated. If the histogram stuff is refactored a bit, `execute_insn_fast` can actually be inlined directly in the single place its used.

That ought to be a future PR though.